### DISCUSSION TITLE
request on some array function

### DISCUSSION THREAD

**Discussion by l1t1** [duckdb/duckdb#5536] [General]
### What happens?

I need union intersect minus operations

### To Reproduce

https://github.com/duckdb/duckdb/discussions/5489

### OS:

Windows

### DuckDB Version:

0.6.0

### DuckDB Client:

CLI

### Full Name:

lutao

### Affiliation:

study

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by Mytherin** [comment]
Please stop opening feature requests as bug reports on the issue tracker. Feature requests should be filed as discussions first. The team can then decide to promote the feature request to an issue if we decide we want to implement this in the short term.

**Comment by l1t1** [comment]
sorry for disturbing.

**Reply by Mytherin** [reply]
That's alright - we have updated the templates to clarify this further now.
